package.json

```
@@ -4,10 +4,11 @@
 	"description": "monorepo for @sveltejs/kit and friends",
 	"private": true,
 	"scripts": {
-		"test": "pnpm test -r --filter=./packages/*",
+		"test": "pnpm test -r --filter=./packages/* --filter=!./packages/create-svelte",
 		"test:cross-platform:dev": "pnpm run --dir packages/kit test:cross-platform:dev",
 		"test:cross-platform:build": "pnpm run --dir packages/kit test:cross-platform:build",
 		"test:vite-ecosystem-ci": "pnpm test --dir packages/kit",
+		"test:create-svelte-templates": "pnpm run --dir packages/create-svelte test",
```

what do you think about shortening this to just `create-svelte`?
```suggestion
		"test:create-svelte": "pnpm run --dir packages/create-svelte test",
```

---

I did actually write it as just `test:create-svelte` at first but this is more descriptive so would prefer to keep it.

---

Another reason I suggested it is that it's easier to remember since it aligns with the project name and easier to type since it's shorter